system/camerad/cameras/spectra.cc

```diff
@@ -172,7 +165,6 @@ void SpectraCamera::sensors_i2c(const struct i2c_random_wr_payload* dat, int len
   int ret = device_config(sensor_fd, session_handle, sensor_dev_handle, cam_packet_handle);
   if (ret != 0) {
     LOGE("** sensor %d FAILED i2c, disabling", cc.camera_num);
-    enabled = false;
```

Comment:
the camera will stay running now?

Reply:
Yes, should we  add an assert here ? After this error, there's no reason to keep camerad running, so a restart may be the best option.

Reply:
`camerad` should always output as many frames as it can